### DISCUSSION TITLE
Pnp search results for only one specific library

### DISCUSSION THREAD

**Discussion by supreet-bajaj** [microsoft-search/pnp-modern-search#3098] [General]
Hi, 

Is it possible to utilize the PnP Search web part to search the content of a specific document library using its global managed metadata properties in SharePoint online 

Thanks,
Supreet Bajaj

**Comment by kasperbolarsen** [comment]
Hi, the answer is most likely a yes, as the path property can limit the search to a specific library. The metadata are commonly used as refiners.
I would be able to be more specific if you can share what you are aiming to achieve and the metadata you have.

**Comment by supreet-bajaj** [comment]
Hi Kasper,

thanks for your response.
I have tried with path property in search query text but it’s showing library title as well in search results.

Could you please help me how I can exclude library name result from it.

{searchTerms}path: library url

**Reply by kasperbolarsen** [reply]
Try something like AND contentclass:"STS_ListItem".

I am on vacation and can't get to my notes, so it is according to my memory 🙃

**Comment by supreet-bajaj** [comment]
Thanks Kasper for your quick response.
Could you please help me to understand why card layout is not showing thumbnail in search results.

or is there any way we can use only one picture for all search results as thumbnail?Attached screenshot is for your reference 
<img width="419" alt="IMG_5629" src="https://github.com/microsoft-search/pnp-modern-search/assets/25117777/02746abc-fd72-442e-9110-b841c459a0b4">
